src/app/system/audit-trails/view-audit/view-audit.component.html

```diff
@@ -0,0 +1,104 @@
+<div class="container">
```

Comment:
50:50 view doesn't seem to look very good. Either we can have both the card and table side by side for larger screens as we have a lot of space. Or you can follow the multiple-column approach for the card as in `/accounting/journal-entries/transactions/view/id` whichever looks better.